mmk_default_rules_alpha.mms

```diff
@@ -195,6 +195,47 @@ GENCAT	    = GENCAT
 GENCATFLAGS =
 
 
+!SDL_UNSUPPORTED_LANGUAGES=DTR:.? DUMP:.? UIL:.UIL?
+!SDL_LANGUAGES=ADA:.ADA BASIC:.BAS BLISS:.R32 BLISS64:.R64 CC:.C DCL:.COM -
+!	FORTRAN:.FOR MACRO:.MAR PASCAL:.PAS PLI:.PL1 SDML:.SDML -
+! 	TPU:.TPU
+!$(FOREACH LANG,$(SDL_LANGUAGES),-
+!.SDL$(LASTWORD $(PATSUBST :, ,$(LANG))) : ; -
+!$(SDL) $(SDLFLAGS)/LANG=$(FIRSTWORD $(PATSUBST :, ,$(LANG)))=$(MMS$TARGET) $(MMS$SOURCE))
```

Comment:
I just realised that the funtion will, of course, produce one looooong line for all the rules.  D'uh.
So we need the multiline macro definition capability first...